packages/knip/fixtures/plugins/biome/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@plugins/biome",
+  "module": "index.ts",
+  "type": "module",
+  "private": true,
+  "devDependencies": {
+    "@biomejs/biome": "^1.9.4",
+    "@types/bun": "latest",
+    "knip": "^5.59.1",
```

Comment:
Do we need all these dependencies here? Let's try to keep fixtures as small and focused as possible.